# 周期
hz: 10

# デバッグ用
test_show: true
sleep_time: 500000000

# 経由点
way_points_x: [0.0, 9.81, 16.3,16.52, 16.58, 10.45, 0.0, -7.65, -17.5, -17.4, -17.19, -9.36,0.0]
way_points_y: [0.0, 0.0, 0.0,7.31, 14.38, 14.40, 14.16, 13.92, 13.76, 6.78, 0.0, 0.0,0.0]

# 車両マージン
margin: 0.25
